### PR TITLE
docker: utilize --cidfile when creating containers

### PR DESCRIPTION
**Pull request by TomasTomecek**
Additional patches:
* probe: log result
* tests,docker: use delete force
* bump dockerfile to F27
* utilize conu container image (from master branch)

### CODE REVIEW

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: failure, return code = __1__ 
* Result url: __[https://da.gd/qKuPX](https://da.gd/qKuPX)__
* Comment: __See details for logs__

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: failure, return code = __1__ 
* Result url: __[https://da.gd/5BhAt](https://da.gd/5BhAt)__
* Comment: __See details for logs__

**Comment by TomasTomecek** [comment]
The tests are now passing for me locally.

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: success, return code = __0__ 
* Result url: __[https://da.gd/70bmN](https://da.gd/70bmN)__
* Comment: __See details for logs__

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: success, return code = __0__ 
* Result url: __[https://da.gd/KvlWw](https://da.gd/KvlWw)__
* Comment: __See details for logs__

File: conu/backend/docker/image.py [hunk 1]
```diff
@@ -153,13 +155,36 @@ def mount(self, mount_point=None):
         """
         return DockerImageFS(self, mount_point=mount_point)
 
+    def _run_container(self, run_command_instance, callback):
+        """ this is internal method """
+        tmpdir = mkdtemp()
+        tmpfile_fd, tmpfile = mkstemp(dir=tmpdir)
+        # the cid file must not exist
+        os.unlink(tmpfile)
```
Comment (dhodovsk) [inline conu/backend/docker/image.py]: The file doesn't exists in newly created tmp directory anyway. Could we create only one temporary directory (value in backend or global), that is here created only if it doesn't exist previously? (one directory for all containers in session)

Comment (TomasTomecek) [reply conu/backend/docker/image.py]: good idea, I'm on it

Comment (TomasTomecek) [reply conu/backend/docker/image.py]: Fuf, this was way difficult than I'm willing to admit.

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: success, return code = __0__ 
* Result url: __[https://da.gd/sWbU](https://da.gd/sWbU)__
* Comment: __See details for logs__

**Pull request merged**